### ISSUE TITLE
Does this work for WEB?

### ISSUE THREAD

**Issue by abutizi** [ionic-team/capacitor-barcode-scanner#14]
Does this work for WEB?

**Comment by abutizi** [comment]
Yes? no? 
Maybe so, 
lets change up the flow, 
make this community grow, 
so that humanity can glow :)

**Comment by rpeight8** [comment]
> Does this work for WEB?

It works

**Comment by mburger81** [comment]
@rpeight8 are you sure it works, for me it doesnt work and I've the same issue as described here #15

**Comment by rpeight8** [comment]
> @rpeight8 are you sure it works, for me it doesnt work and I've the same issue as described here #15

Yeah, I’m using `@capacitor/barcode-scanner` version 1.0.1 along with Capacitor v6 (6.0.0). I also have several other packages installed, which might be causing some side effects.
Anyway, I've previously tested it on both Web (Chrome) and Android platforms... 

You can reach out to me on Discord, and we can try to figure something out: rpeight8

**Comment by Scooby27** [comment]
@rpeight8 @mburger81 did you manage to prove it working on web? I have the same as #15 on web only
